src/index.js

```diff
@@ -686,14 +686,48 @@ PubSub.prototype.request = function(config, callback) {
 
   if (!gaxClient) {
     // Lazily instantiate client.
-    gaxClient = v1(this.options)[config.client](this.options);
+    if (config.client === 'publisherClient') {
```

Comment:
I think it would be better if we went over the places we call `this.request()` and changed `publisherClient` or `subscriberClient` to the correct casing of the class. We used the lowercase to match the class name at the time, but if that's changing, we should change how we refer to it as well.

Reply:
Updated the case to PublisherClient and SubscriberClient.